.gitignore

```diff
@@ -82,9 +82,6 @@ dist
 # Editor/IDE
 .idea
 .cursor/*
-.vscode/*
-#!.vscode/settings.json
-!.vscode/spellright.dict
```

Comment:
I removed .vscode from .gitignore because the contents of this folder are meant to be shared. Specifically, .vscode/settings.json holds Workspace Settings, which are designed for the whole team. This is different from User Settings, which live outside the repo and should remain personal.